azurerm/resource_arm_template_deployment.go

```diff
@@ -42,6 +42,11 @@ func resourceArmTemplateDeployment() *schema.Resource {
 				Optional: true,
 			},
 
+			"parameters_body": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Can you set this as mutually exclusive with `parameters` (see `ConflictsWith`) and validate that it is valid JSON?

Reply:
You may also want a `StateFunc` to normalize the JSON so that things like whitespace don't cause diffs.

Reply:
pushed a commit to add these